src/patient-search/patient-search.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { doSearch } from "./patient.resource";
-import { css } from "@emotion/core";
-
+// @ts-ignore
```

Comment:
You can get typescript to be okay with css files. Inside of the src directory, create a file like this: https://github.com/joeldenning/openmrs-esm-navbar/blob/master/src/declarations.d.ts

Reply:
done